.releaserc

```diff
@@ -0,0 +1,6 @@
+{
+  "release": {
+    "success": false,
+    "fail": false
+  }
+}
```

Comment:
What's this?

Reply:
See: https://github.com/semantic-release/semantic-release/releases/tag/v14.0.0